VRF V2 Direct funding docs (#854)
smartcontractkit/documentation: docs/Using Randomness/v1/get-a-random-number.md

The contract should own enough LINK to pay the specified fee. The beginner walkthrough explains how to [fund your contract](/docs/fund-your-contract/).

> Note, the below values have to be configured correctly for VRF requests to work. You can find the respective values for your network in the [VRF Contracts page](/docs/vrf/v1/configuration/).

Reviewer: ```suggestion
Note, the following values must be configured correctly for VRF requests to work. You can find the respective values for your network in the [VRF Contracts page](/docs/vrf/v1/configuration/).
```
